src/promptflow/promptflow/_sdk/operations/_trace_operations.py

```diff
@@ -155,3 +158,48 @@ def _list_line_runs_with_runs(self, runs: typing.List[str]) -> typing.List[LineR
                 if line_run is not None:
                     line_runs.append(line_run)
         return line_runs
+
+    @monitor_operation(activity_name="pf.traces.delete", activity_type=ActivityType.PUBLICAPI)
+    def delete(
+        self,
+        run: typing.Optional[str] = None,
+        session: typing.Optional[str] = None,
+        started_before: typing.Optional[str] = None,
```

Comment:
NOT SURE IF WE SHOULD PROVIDE some systax sugar to say something like 1 day ago

Reply:
thumb up as users, but thumb down as a dev... will draft some proposals on this